### ISSUE TITLE
A11y_AzureToolsStorageExplorer_BlobOperations_ClonewithNewNameBlob_ColorContrast: "i" non text content does not meet minimum color contrast ratio of 3.5:1.

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/AzureStorageExplorer#2611]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11YTCS; #DesktopApp; #A11yUsable; #Mac; #Benchmark; #ColorContrast

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: Version: 1.12.0
MacOS: Catalina 10.15.3

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Once completed, Navigate to "Upload" button and Upload a blob/file by using  "Upload Files" control.
6. Once Completed, Right click on uploaded blob and Select "Clone with New Name" and hit enter.
7. Navigate to "i" non text and check the color contrast

​**Actual:**
​"i" non text content does not meet minimum color contrast ratio of 3.5:1.

**Expected:**
Non text content should maintain minimum color contrast ratio of 3.5:1.

**User Impact:**
Low vision users will have difficulty in viewing the non text element.

**Comment by Shivamd1** [comment]
We have uploaded repro video of this bug in [OneDrive](https://microsoft-my.sharepoint.com/:f:/p/v-subha/EvxHdg1zkN9PknF1uOxnOiQBejpxKYg9n21DydgQ_om0ew?e=WK2Zih) link with name "2611_A11y_AzureToolsStorageExplorer_BlobOperations_ClonewithNewNameBlob_ColorContrast" please refer the Screenshot for reference.
Please let us know if there is any problem in accessing the link

**Comment by MRayermannMSFT** [comment]
@Shivamd1 the link provided does not work. I get a 404 error.

**Comment by Shivamd1** [comment]
I have uploaded repro video of this bug in OneDrive [link](https://microsoftapc-my.sharepoint.com/:i:/g/personal/v-shprak_microsoft_com/EVpL8qxKlk1EsiywK650g8QBLp4_Y5dPgfDeipWhnzDdfw?e=ypc3E7) with name "2611_A11y_AzureToolsStorageExplorer_BlobOperations_ClonewithNewNameBlob_ColorContrast" please refer the Screenshot for reference.

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Mac-Sep2019;

**Comment by kapilvaishna** [comment]
Closing the bug as this is a non SFP product and the bugs is filed before 2022 under Sev4
#Closed;

**Comment by charan-kamasani** [comment]
Closing per team decision: Sev4 accessibility bug filed before 2022 on a non-SFP product. The UI has since been updated and this specific color contrast issue is no longer being tracked.
